connect/runtime/src/main/java/org/apache/kafka/connect/runtime/distributed/DistributedConfig.java

```diff
@@ -38,6 +38,7 @@
      */
     public static final String GROUP_ID_CONFIG = "group.id";
     private static final String GROUP_ID_DOC = "A unique string that identifies the Connect cluster group this worker belongs to.";
+    private static final String GROUP_ID_CONFIG_DEFAULT = "connect";
```

Comment:
I wonder if it's a good idea to have a default for this and the storage topics below. It's convenient when getting started, but it also increases the chances for conflict since users might not know it needs to be unique. Thoughts?